bedrock/utils/migrations/0003_auto_20190710_1159.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.3 on 2019-07-10 18:59
+
+from django.db import migrations
+
+
+def fix_bytes_repr(apps, schema_editor):
+    """Values were stored as strings with the "b''" in them. This should fix that."""
+    GitRepoState = apps.get_model('utils', 'GitRepoState')
+    for repo in GitRepoState.objects.all():
+        if repo.latest_ref.startswith("b'"):
+            # strip the b' off the start and the ' off the end
+            repo.latest_ref = repo.latest_ref[2:][:-1]
```

Comment:
I decided against that for a few reasons:

* I don't see any reason for an entry to start with `b'` and not end with a single quote.
* These values will be updated and replaced the next time these repos are updated anyway.
* We only really use the first 10 characters for the most part.
* Even if we did lop off a character of the hash it would just cause the repo to pull changes, so not a big deal.

This really wasn't even necessary as it would eventually fix itself once all of the repos updated. So this is just me wanting it to be fixed sooner than later :)